src/docs/guides/24-how-to-use-time-based-bookings/index.md

```diff
@@ -0,0 +1,131 @@
+          {dayOfWeek: 'wed', seats: 1, startTime: '09:00', endTime: '11:00'},
+          {dayOfWeek: 'wed', seats: 1, startTime: '12:30', endTime: '16:30'},
+          {dayOfWeek: 'thu', seats: 1, startTime: '09:00', endTime: '17:00'},
+          {dayOfWeek: 'fri', seats: 1, startTime: '09:00', endTime: '17:00'},
+          {dayOfWeek: 'sat', seats: 1, startTime: '10:00', endTime: '15:00'},
+        ],
+      },
+      publicData: {
+        address: {
+          city: 'New York',
+          country: 'USA',
+          state: 'NY',
+          street: '230 Hamilton Ave',
+        },
+        category: 'road',
+        gears: 22,
+        rules: 'This is a nice, bike! Please, be careful with it.',
+      },
+      price: new Money(2290, 'USD'),
+      images: [new UUID('5caede66-e4e2-4f5b-90e4-9e3f58fc7fd8')],
+    },
+    {
+      expand: true,
+      include: ['images'],
+    }
+  )
+  .then(response => {
+    // handle response
+  });
+```
+
+For more information about the plan attributes, see
+[the API documentation](https://www.sharetribe.com/api-reference/#ownlisting-availability-plan).
+By default FTW defines a day-based plan for all listings in the
+[EditListingAvailabilityPanel](https://github.com/sharetribe/flex-template-web/blob/master/src/components/EditListingAvailabilityPanel/EditListingAvailabilityPanel.js)
+component.
+
+As for creating time-based bookings, they are created just like any
+other booking. The only limitations for start and end times are that the
+end needs to be after the start and they both need to be divisible by 5
+minutes.
+
+## 3. A few things to consider
+
+### Transaction processes and listing availability plans
+
+A time-based availability plan needs to be explicitly declared for a
+listing as a default booking type can not be defined for a marketplace.
+The booking type is defined by the transaction process and a single
+marketplace can have multiple processes which can have different booking
+types. Listings are not tied to any booking process and this should be
+kept in mind when creating a booking for a listing in case your
+marketplace supports day- and time-based bookings. In this kind of
+scenario it is wise to store the listing booking type in the listing's
+[public data](/references/extended-data), so that the correct
+transaction process can be resolved when booking each listing.
+
+### UI changes for time-based bookings
+
+To effectively use time-based bookings, a UI is required for managing
+the availability plan and exceptions of a listing and for defining the
+booking length.
+
+Unfortunately, at the moment FTW does not provide any UI for creating
+bookings shorter than one day or for managing time-based availability
+exceptions or plans. So in order to get the benefits of time-based
+bookings you will need to implement these missing UIs as they are
+required by your marketplace.
+
+In the future we plan to ship a wider range of components with the FTW
```

Comment:
"In the future, we plan to ship a template app that covers time-based bookings too."

We are likely to create another template app (a fork from FTW), but let's see how the discussion/planning goes with this epic.

Reply:
Suggestion by @Gnito makes sense. A couple of smaller style-related points related to this part:
- Let's remove "unfortunately". It's sounds a bit too much like we're apologizing for not having more stuff. I don't think there's need for that.
- Let's talk about "user interface" instead of UI.
- In each article, the first instance of FTW should be Flex Template for Web (FTW). Of course most people reading this particular article will know the meaning of the acronym, but it's still a good practice to assume people reading a particular article don't know all the acronyms, so they should be spelled out

So, let's replace this paragraph: "Unfortunately at the moment FTW does not provide any UI for creating
bookings shorter than one day or for managing time-based availability exceptions or plans. So in order to get the benefits of time-based bookings you will need to implement these missing UIs as they are required by your marketplace."

Let's have this paragraph instead: "At the moment, Flex Template for Web (FTW) does not provide an example user interface for initiating a booking shorter than one day or for managing time-based availability
exceptions or plans. Thus, in order to use time-based bookings, you will need to implement these parts of the user interface yourself."